webpack.config.js

```diff
@@ -18,12 +18,26 @@ module.exports = {
         use: { loader: 'babel-loader' },
       },
     ],
+    loaders: [
+      { exclude: ['node_modules'], loader: 'babel', test: /\.jsx?$/ },
```

Comment:
I know you talked about this at standups a little, but would you be able to do a little share about how you got it configured? EIther at next standups or if I'm the only one interested whenever you have some time. I'd love to learn.